a3p-integration/proposals/c:price-feeds/priceFeed.test.js

```diff
@@ -0,0 +1,121 @@
+import test from 'ava';
+
+import {
+  agd,
+  agops,
+  evalBundles,
+  executeOffer,
+  getVatDetails,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+  newOfferId,
+  waitForBlock,
+} from '@agoric/synthetic-chain';
+
+const SUBMISSION_DIR = 'priceFeed-submission';
+
+const ORACLE_ADDRESSES = [GOV1ADDR, GOV2ADDR, GOV3ADDR];
+
+test('update all priceFeed vats', async t => {
```

Comment:
I don't see where this test updates anything.

Is it testing that something was updated?

Reply:
line 22 evals `priceFeed-submission` 

@Chris-Hibbert is `updatePriceFeeds.js` meant to be its own proposal or part of an upgrade? If it's a CORE_EVAL proposal then make the proposal package type be like,
```
  "agoricProposal": {
    "type": "/agoric.swingset.CoreEvalProposal",
    "source": "subdir",
```
`b:localchain` is a good example.

If it will be part of an Software Upgrade, then put it in `a:upgrade-next` and its Go app handler like the other proposals there. 'upgrade-next' has a mix of u14 and u15 so we'll already be removing the u14 proposals once that hits Mainnet.